src/app/components/home-page-container/home-page-container.component.sass

```diff
@@ -0,0 +1,148 @@
+$section-left-margin: 10%
+$subtle-pink: #F2E3D5
+$deep-blue:#08348C
+$light-blue: #07418C
+$orange: #F24405
+$bright-orange: #F27830
+$shadow-color: rgba(0, 0, 0, 0.2)
+$background:#F5F1ED
+// $shadow-color: #F2E3D550
+
+#main-image
+    position: absolute
+    right: 0
+    bottom: 0
+    height: 70%
+
+.shadow
+    -moz-box-shadow:    inset 0 0 50px 20px $shadow-color
+    -webkit-box-shadow: inset 0 0 50px 20px $shadow-color
+    box-shadow:         inset 0 0 50px 20px $shadow-color
+
+.page-container
+    height: 100%
+    width: 100%
+
+.home-page-container
+    @extend .page-container
+    background-color: $background
+
+
+.second-page-container
+    @extend .page-container
+    display: flex
+    flex-direction: column
+    justify-content: center
+    align-items: center
+    background-color: $deep-blue
+
+.third-page-container
+    @extend .page-container
+    display: flex
+    flex-direction: column
+    justify-content: center
+    align-items: center
+    background-color: $bright-orange
+
+.fourth-page-container
+    @extend .page-container
+    display: flex
+    flex-direction: column
+    background-color: $light-blue
+
+
+.final-page-container
+    @extend .page-container
+    display: flex
+    flex-direction: column
+    background-color: $bright-orange
+
+
+.home-page-title
+    position: relative
+    top: 45%
+    left: 15%
+
+.header
+    font-size: 60px
+    font-weight: 900
+    color: $deep-blue
+
+.subheader
+    font-size: 32px
+    font-weight:600 
+    margin-top: 30px
+    color: $light-blue
+        
+        
+.button
+    background-color: $orange
+    border-radius: 10px
+    color: white
+    font-size: 20px
+    margin-top: 30px
+
+.button-white
+    background-color: $subtle-pink
+    color: $orange
+    border-radius: 10px
+    font-size: 20px
+    margin-top: 30px
+
+.navigation
+    position: absolute
+    bottom: 0
+    left: 0
+    width: 100%
+    height: 5%
+    display: flex
+    justify-content: center
+    align-items: center
+
+    img
+        height: 100%
```

Comment:
@neshang  you had `fill: purple !important` here before, but I don't think it was working, because it is in an `<img>` element instead of an inline `<svg>`... It's just a minor thing, and black looks good anyhow, so we can look at this again in a future PR if we have time :)